### DISCUSSION TITLE
NPM shows an incorrect TS/DT badge for an old package version

### DISCUSSION THREAD

**Discussion by hheimerd** [npm/feedback#1101] [Website] [answered]
I tried to check TypeScript support for a specific version of Prettier. When I opened the version history, NPM showed me that any version supported TypeScript. This is incorrect. I think the website is showing the latest version badge for all versions of the package.

https://www.npmjs.com/package/prettier/v/0.0.3
<img width="374" alt="Screenshot 2024-04-03 at 15 33 06" src="https://github.com/npm/feedback/assets/38163258/2d084607-0c26-4756-9bcc-71e0a36eef8e">

**Comment by leobalter** [comment, marked as answer]
Thanks for the heads up! Our website currently limits some information - including the TS badge - being related only to the latest version published.

**Reply by ljharb** [reply]
It makes sense to only display it *on* the latest version, but it's confusing if it's shown on a different version page. Could information in this bucket (only for the latest) be hidden when not showing the latest version?

**Reply by leobalter** [reply]
I agree it can be improved and this something we should keep in our radar.
